manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.5.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.prusa3d.com/downloads/drivers/PrusaSlicer_Win_standalone_2.5.0.exe
+  InstallerSha256: 1A417FE95D2EEBD7E0D5A4A2A51E043BA594A865F223841A45CA6D4E88E9444C

```

Maybe it was cached?

```
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
Expected hash: 1a417fe95d2eebd7e0d5a4a2a51e043ba594a865f223841a45ca6d4e88e9444c
```
```
Get-FileHash ./PrusaSlicer_Win_standalone_2.5.0.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
```

```suggestion
  InstallerSha256: F97CAAE8172E3204538E855EE9A32F04FC41BE0D67FF7E54C0C903E6DC1DBC26
```